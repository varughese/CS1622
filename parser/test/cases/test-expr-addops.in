void main(void)
{
    1 + 2;
    1 - 2;
}
