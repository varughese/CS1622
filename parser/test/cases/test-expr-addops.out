[program
  [fun-declaration
    [void]
    [main]
    [params]
    [compound-stmt 
      [+  [1] [2]]
      [-  [1] [2]]
    ]
  ]
]
